recipe/meta.yaml

```diff
@@ -1,46 +1,87 @@
 {% set name = "pyFFTW" %}
 {% set version = "0.13.1" %}
-{% set sha256 = "09155e90a0c6d0c1f2d1f3668180a7de95fb9f83fef5137a112fb05978e87320" %}
+{% set sha256 = "ffc1b3e7f4f74ee38716d668d7606b5fbfded58343655c160c3644fc4acf5d55" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  url: https://github.com/{{ name }}/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
   sha256: {{ sha256 }}
   patches:
-    - no-system-include.patch
+    - patches/0001-cython3-updates-and-test-build-linker-flags.patch
+    - patches/0002-no-system-include.patch
 
 build:
   number: 0
-  skip: true  # [py<39 and arm64]
+  # fftw not available for s390x
+  skip: true  # [py<38 or py>311 or s390x]
```

Comment:
Does the build fails on 3.12?

Reply:
`versioneer` is not updated, 

```
  Running command Preparing metadata (pyproject.toml)
  /opt/conda/conda-bld/pyfftw_1710867425879/work/versioneer.py:421: SyntaxWarning: invalid escape sequence '\s'
    LONG_VERSION_PY['git'] = '''
  Traceback (most recent call last):
    File "/opt/conda/conda-bld/pyfftw_1710867425879/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh/lib/python3.12/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py", line 353, in <module>
      main()
    File "/opt/conda/conda-bld/pyfftw_1710867425879/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh/lib/python3.12/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py", line 335, in main
      json_out['return_val'] = hook(**hook_input['kwargs'])
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    File "/opt/conda/conda-bld/pyfftw_1710867425879/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh/lib/python3.12/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py", line 149, in prepare_metadata_for_build_wheel
      return hook(metadata_directory, config_settings)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    File "/opt/conda/conda-bld/pyfftw_1710867425879/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh/lib/python3.12/site-packages/setuptools/build_meta.py", line 396, in prepare_metadata_for_build_wheel
      self.run_setup()
    File "/opt/conda/conda-bld/pyfftw_1710867425879/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh/lib/python3.12/site-packages/setuptools/build_meta.py", line 341, in run_setup
      exec(code, locals())
    File "<string>", line 818, in <module>
    File "<string>", line 757, in setup_package
    File "/opt/conda/conda-bld/pyfftw_1710867425879/work/versioneer.py", line 1480, in get_version
      return get_versions()["version"]
             ^^^^^^^^^^^^^^
    File "/opt/conda/conda-bld/pyfftw_1710867425879/work/versioneer.py", line 1412, in get_versions
      cfg = get_config_from_root(root)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^
    File "/opt/conda/conda-bld/pyfftw_1710867425879/work/versioneer.py", line 342, in get_config_from_root
      parser = configparser.SafeConfigParser()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  AttributeError: module 'configparser' has no attribute 'SafeConfigParser'. Did you mean: 'RawConfigParser'?
  error: subprocess-exited-with-error

  × Preparing metadata (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> See above for output.

```

See for example (another package): https://github.com/dask/dask-image/pull/357/files

and when I tried to update it, there were

 other problems:

```
pyfftw/pyfftw.c:1205:10: fatal error: numpy/arrayobject.h: No such file or directory
```

for future reference here the patch to update `versioneer`: [0003-update-versioneer-for-py312-support.patch](https://github.com/AnacondaRecipes/pyfftw-feedstock/files/14656175/0003-update-versioneer-for-py312-support.patch)

maybe solvable https://stackoverflow.com/questions/14657375/cython-fatal-error-numpy-arrayobject-h-no-such-file-or-directory

but, SF does not need py312 necessarily. I would keep things as they are.